octopus_sensing/devices/skinos_streaming.py

```diff
@@ -0,0 +1,264 @@
+# This file is part of Octopus Sensing <https://octopus-sensing.nastaran-saffar.me/>
+# Copyright © Nastaran Saffaryazdi 2020
+#
+# Octopus Sensing is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software Foundation,
+#  either version 3 of the License, or (at your option) any later version.
+#
+# Octopus Sensing is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with Octopus Sensing.
+# If not, see <https://www.gnu.org/licenses/>.
+
+import os
+import platform
+import threading
+import datetime
+import csv
+import math
+import struct
+import serial
+from typing import List, Optional
+import time
+import socket
+
+from octopus_sensing.devices.monitored_device import MonitoredDevice
+from octopus_sensing.common.message_creators import MessageType
+from octopus_sensing.common.message import Message
+from octopus_sensing.devices.common import SavingModeEnum
+
+HOST = '127.0.0.1'
+PORT = 50007
+
+class SkinosStreaming(MonitoredDevice):
+
```

Comment:
Can you please add a document for the class, explaining what device is this, and what parameters the constructor accepts?
These documents will automatically add to Octopus Sensing's API Docs.